metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/validator.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from typing import List, Optional, Tuple
+
+from lark import Tree
+
+from datahub.ingestion.source.powerbi.m_query import resolver, tree_function
+
+LOGGER = logging.getLogger(__name__)
+
+
+def any_one_should_present(
```

Comment:
any_function_should_be_known -> function should be named consistently with the one below.

Reply:
I removed these functions, they were not utilized.